pkg/operator/apiserver/controller/workload/workload.go

```diff
@@ -273,6 +273,34 @@ func (c *Controller) updateOperatorStatus(workload *appsv1.Deployment, operatorC
 		Status: operatorv1.ConditionFalse,
 	}
 
+	if !preconditionsReady {
+		message := "the operator didn't specify what preconditions are missing"
+		for _, err := range errs {
+			message = message + err.Error() + "\n"
```

Comment:
space missing for the first appended error.

Reply:
should we replace the placeholder message if errs is given ?

Reply:
yes we should